### ISSUE TITLE
Lexicon - Default browser focus indicator on Menu button

### ISSUE THREAD

**Issue by catbusofthecanals** [jadu/pulsar#1579]
**Describe the bug**
At smaller viewports or 200% zoom, the Main/Close (Toggle Main Menu) button displays a browser default focus indicator and not a custom indicator that matches the styling of the other focus indicators displayed on the page.

**To Reproduce**
1. Go to Pulsar Lexicon
2. Set viewport width to under 992 px in width
3. Tab through page with keyboard to the Menu button
4. See default browser indicator

**Expected behavior**
All interactive elements should have a consistent set focus indicator, ideally a two tone 2px solid outline.

**Screenshots**

<img width="716" alt="Image" src="https://github.com/user-attachments/assets/cd5193b2-1357-4812-a994-9f1087f6fbe6" />

**Additional context**
Related success criteria:
[2.4.7 Focus Visible (AA)](https://www.w3.org/WAI/WCAG22/Understanding/focus-visible)
[2.4.13 Focus Appearance (AAA)](https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance)

**Suggested fixes**
There is a CSS class targeting the Menu button but it only indicates border-width. Add styling to the .mobile-menu-button when on focus, so that the focus indicator is inline with the other indicators on the page.

outline: 3px solid #000000;   
background-color: #fd0;   
color: #000;
